scripts/tests/test_asset_drift_check.py

```
@@ -0,0 +1,273 @@
+            {
+                "lexnlp.ml.catalog": catalog_mock,
+                "lexnlp.ml.catalog.download": dl_mock,
+            },
+        ):
+            rc = asset_drift_check.main(["--manifest", str(manifest_path)])
+
+        assert rc == 0
+
+
+# ---------------------------------------------------------------------------
+# load_manifest – basic validation
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)